app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)

```

Comment:
According to the checklist, when importing multiple classes/modules, it's recommended to use parentheses `()` for better readability. Consider changing the import style.